### ISSUE TITLE
Several games do not boot without the following directories

### ISSUE THREAD

**Issue by Morph1984** [yuzu-emu/yuzu#2963]
Directory structures

```
yuzu
└───nand
    └───user
        └───save
            └───cache (does not exist)
                └───0000000000000000 (does not exist)
```

```
yuzu
└───nand
    └───temp (does not exist)
        └───0000000000000000 (does not exist)
            └───00000000000000000000000000000000 (does not exist)
                └───0000000000000000 (does not exist)
```

The directories above are not generated as of the latest builds (canary 2649 and later)

Currently, the known games that need the cache directory are the following:
- Super Mario Maker 2

Currently, the known games that need the temp directory are the following:
- Bloons TD 5
- NES/SNES Online

For the temp directory, **you have to re-create it on every launch** as yuzu deletes this folder on launch.

As a temporary workaround you have to create the folders
"yuzu\nand\user\save\cache\0000000000000000" and
"yuzu\nand\temp\0000000000000000\00000000000000000000000000000000\0000000000000000"

or, you can download this zip and place the folders to match the above directories 
[nand.zip](https://github.com/yuzu-emu/yuzu/files/3746048/nand.zip)

For windows users (who do not have yuzu installed in portable mode) 
run this command in cmd:

```batch
if exist "%APPDATA%\yuzu" (
    if not exist "%APPDATA%\yuzu\nand\user\save\cache\0000000000000000" (
        mkdir "%APPDATA%\yuzu\nand\user\save\cache\0000000000000000"
    )
)
```

The cause is traced to the PR #2430

**Comment by windyboy1704** [comment]
This issue started since the September Patreon, I cannot get SMM2 boot. Thanks for your help

**Comment by Hexagon12** [comment]
The issue no longer appears on the latest mainline builds. Fixed by #3007.
